cognite/neat/_session/_to.py

```diff
@@ -35,5 +37,30 @@ def yaml(self, io: Any | None = None) -> str | None:
 
 
 class CDFToAPI:
-    def __init__(self, client: CogniteClient | None) -> None:
+    def __init__(self, state: SessionState, client: CogniteClient | None, verbose: bool) -> None:
         self._client = client
+        self._state = state
+        self._verbose = verbose
+
+    def instances(self, space: str | None = None):
+        if not self._state.verifies_dms_rules:
+            raise ValueError("No verified DMS data model available")
+
+        space = space or self._state.verifies_dms_rules.metadata.space
```

Comment:
I think we should not do this. Instead lets prompt the user for instance space if they do not set it.

The `metadata.space` is a schema space and should not be used for data.

Reply:
Agree !